.github/workflows/ci.yml

```diff
@@ -56,7 +56,7 @@ jobs:
 
       - name: Test with pytest
         run: |
-          pipenv run pytest
+          pipenv run pytest || [ $? -eq 5 ]
```

Comment:
May I know what is this command? (sorry I'm not good as Bash script 😵‍💫)

Reply:
I tried to explain this change in the message for [the commit](https://github.com/City-Bureau/city-scrapers-template/pull/24/commits/e259c4232d2694c89cbc3552ea19dd94af967316). Would you prefer I add a comment above this step? I was tempted to but I didn't want to add too much noise/fluff to this workflow file.

Reply:
Ah I see. It's clear to me now. You can comment about the `exit code: 5` of `pytest`
https://docs.pytest.org/en/stable/reference/exit-codes.html

> Running pytest can result in six different exit codes:
Exit code 5:No tests were collected

Reply:
I've added a comment, let me know if that works for you, @LienDang